server/server.go

```diff
@@ -285,12 +283,16 @@ func recordLock(inst *instance.Instance, service flux.ServiceID, locked bool) er
 	return nil
 }
 
-func (s *Server) PostRelease(inst flux.InstanceID, spec flux.ReleaseJobSpec) (flux.ReleaseID, error) {
-	return s.releaser.PutJob(inst, spec)
+func (s *Server) PostRelease(inst flux.InstanceID, params jobs.ReleaseJobParams) (jobs.JobID, error) {
+	return s.jobs.PutJob(inst, jobs.Job{
+		Method:   jobs.ReleaseJob,
+		Priority: jobs.PriorityInteractive,
+		Params:   params,
+	})
 }
 
-func (s *Server) GetRelease(inst flux.InstanceID, id flux.ReleaseID) (flux.ReleaseJob, error) {
-	return s.releaser.GetJob(inst, id)
+func (s *Server) GetRelease(inst flux.InstanceID, id jobs.JobID) (jobs.Job, error) {
```

Comment:
Since this is `GetRelease`, shouldn't it check that it's actually a release job?